src/com/rcv/Main.java

```diff
@@ -13,13 +13,16 @@
 
 package com.rcv;
 
+import com.rcv.CVRReader.SourceWithUnrecognizedCandidatesException;
+import com.rcv.CVRReader.UnrecognizedCandidateException;
```

Comment:
This import is unused. Do you have "Optimize Imports" checked in your commit window?